### PR TITLE
Add doc around terra releases.

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary
I added some doc around terra releases. The intention of this is be able to share with consumers asking when something will be released.

Thanks for contributing to Terra. 
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

**Comment by mjhenkes** [comment]
I will add those topics

File: src/terra-dev-site/guide/Releases.c.guide.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Releases
+
+## Release cycle
+* Weekly.
+  * Terra will release any componet that has had changes in the last week. Generally we relase mid week on tuesday or wednesday.
```
Comment (dkasper-was-taken) [inline src/terra-dev-site/guide/Releases.c.guide.md]: 'component'

Comment (dkasper-was-taken) [reply src/terra-dev-site/guide/Releases.c.guide.md]: 'release'

File: src/terra-dev-site/guide/Releases.c.guide.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+## Bugs
+* We aim for zero net new bugs and a turn around time of 30 days.
+
+## When will X issue be fixed?
+As a widely used framework our first concern is quality, all changes to terra must meet our standards before we release them. That said there are a few indicators to help you predict when a change could be available.
+
+1. Does the issue have linked pull request?
+  * Most pull requests are completed within 1 to 2 weeks of being issued and released within a week.
+2. Second, check for your issue on our [project board](https://github.com/orgs/cerner/projects/1)
+  * If the issue is there it means it’s currently being worked on and you can follow the progress.
```
Comment (yuderekyu) [inline src/terra-dev-site/guide/Releases.c.guide.md]: nit: you can *track progress

File: src/terra-dev-site/guide/Releases.c.guide.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+3. If the issue is neither out for pull request or in process, it’s been prioritized below other issues
+  * To express interest in the issue add a comment to the issue and if there is enough interests we may prioritize the issue higher.
```
Comment (yuderekyu) [inline src/terra-dev-site/guide/Releases.c.guide.md]:
```suggestion
  * To express interest in the issue add a comment to the issue. If there is enough interest we may prioritize the issue higher.
```

File: src/terra-dev-site/guide/Releases.c.guide.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Releases
+
+## Release cycle
+
+* Weekly.
+  * Terra will release any component that has had changes in the last week. Generally we release mid week on Tuesday or Wednesday.
```
Comment (bjankord) [inline src/terra-dev-site/guide/Releases.c.guide.md]: nit: I'd update this to `Terra will release any component that has had changes reviewed and merged into master the last week`

File: src/terra-dev-site/guide/Releases.c.guide.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+* One-offs released as needed.
+  * We keep our master branch in an always release-able state to enable quick turnaround time.
+
+## Bugs
+
+* Terra aims for zero net new bugs and a turn around time of 30 days.
+
+## Versioning
+
+* Terra uses [Semantic Versioning](https://semver.org/) to indicate changes in our javascript api.
```
Comment (bjankord) [inline src/terra-dev-site/guide/Releases.c.guide.md]: nit: change `javascript` to `JavaScript` and `api` to `API`

File: src/terra-dev-site/guide/Releases.c.guide.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+## When will X issue be fixed
+
+As a widely used framework our first concern is quality, all changes to terra must meet our standards before we release them. That said there are a few indicators to help you predict when a change could be available.
+
+1. Does the issue have a linked pull request?
+  * Most pull requests are completed within 1 to 2 weeks of being issued and released within a week.
+2. Second, check for your issue on our [project board](https://github.com/orgs/cerner/projects/1).
+  * If the issue is there it means it’s currently being worked on and you can track the progress.
+3. If the issue is neither out for pull request or in process, it’s been prioritized below other issues.
+  * To express interest in the issue add a comment to the issue If there is enough interest we may prioritize the issue higher.
```
Comment (bjankord) [inline src/terra-dev-site/guide/Releases.c.guide.md]: Some punctuation updates.
```suggestion
  * To express interest in the issue, add a comment to the issue. If there is enough interest, we may prioritize the issue higher.
```

**Pull request merged**